app/config/settings_prod.py

```diff
@@ -274,3 +274,7 @@ def get_logging_config():
 # By default django_promtheus tracks the number of migrations
 # This causes troubles in various places so we disable it
 PROMETHEUS_EXPORT_MIGRATIONS = False
+
+
+# Where the application goes to find its static files.
+STATIC_URL = "/static/"
```

Comment:
The `STATIC_URL` is already defined in line 178 and has been modified by  PR #192

Reply:
and now we've removed it.